### ISSUE TITLE
Champion Planning – Cohort 1 

### ISSUE THREAD

**Issue by jules32** [Openscapes/leadership#4]
The ultimate vision of Openscapes is that we can get to environmental solutions faster if we are more efficient and collaborative in how we do science. 

The Openscapes Champion* program is a mentor program for early career researchers. Its main goal is to help Champions and their lab** work more efficiently using existing open tools. The program will be a mix of one-on-one meetings with Champions and Julie, and community calls with all Champions and members from their labs. This way, it's possible to discuss and plan specifically for individual Champions, have concrete progress from their labs, and connect and learn with other Champions and their labs. This will strengthen the community of practice around open, collaborative environmental science. 

*name may change
**“Lab” is broadly defined; it’s whatever group you work with or will be working with in the future. It can even be Future You.

**Comment by jules32** [comment]
## Correspondence Summary

- [x] Sept 28. Email sent to pilot cohort to introduce the idea of Openscapes and invite participation 
    - [x] participation confirmed
- [x] Nov 2. Emails sent to individual Champions with more details and asking to book an appointment slot for November call

**Comment by jules32** [comment]
# Want to be a part of the Openscapes pilot cohort?
*Emailed Sept 28, lightly edited*

I have a [Mozilla Fellowship](https://blog.mozilla.org/blog/2018/08/21/mozilla-announces-25-new-fellows-in-openness-science-and-tech-policy/) to develop what I am calling “Openscapes” a program at NCEAS to "open up the landscapes of ecology & environmental science through open communities, data, and code". I want to increase the value and practice of open data science in the environmental sciences, basically building from [our Nature paper](https://www.nature.com/articles/s41559-017-0160). I’m super excited! I want being a part of Openscapes to be something you’d want to put on your CV and help you do your awesome science, and help us all get to environmental solutions faster! 

As part of this, I want to have cohorts of “Champions” (naming may change), with you wonderful people being in the pilot cohort that I engage with 1:1 through ~June. I am still working out what this will mean exactly (with Mozilla’s help) but will 100% be respectful of your time. I want to identify places to help and support you and your labs to be more efficient with analyzing and collaborating with data. I’m envisioning a few calls with you, hopefully calling in remotely to a lab meeting, and would love to meet your labs in-person if we can swing it. 

I will follow up with each of you individually more, I know I have talked about this stuff with you in varying stages and this is still quite informal but I wanted to get the ball rolling. I’ve already been talking with some of you about all this stuff – with awesome outcomes: a new course for Efficient Computational Skills (woot!) and switching from Matlab to R, and joining twitter to learn from the #rstats community (woot!).

I know this is a lot, but, I am super excited to work with you all and figure out how I can help you. 

**Next steps:** please let me know if you’re interested! I won’t get anything formally going until November

**Comment by jules32** [comment]
# Openscapes Planning – Name
*Emailed Nov 2, lightly edited*

TL;DR: I’m excited to start working with you through Openscapes! The program will be from Jan-May and be done remotely. Please book an hour call with me in November to discuss. 

**Openscapes Plan**
I am so excited to work with you through Openscapes! I have been formalizing the plan for the program that will be from Jan-May 2019. And I am ready to meet with you to introduce my ideas and discuss your needs.


The ultimate vision of Openscapes is that we can get to environmental solutions faster if we are more efficient and collaborative in how we do science. The main goal of the “Championship” program is to help you and your lab* work more efficiently using existing open tools – while engaging you but requiring the least amount of time from you personally. The program will be a mix of one-on-one meetings with you and me, and community calls with all Champions and members from their labs. This way, we’ll be able to discuss and plan specifically for you, have concrete progress from your lab, and connect and learn with other Champions and their labs. 


**Details & Timeline for 2019**
We will have 3 scheduled meetings (total of 4 hours) each month from Jan-May 2019. (Yes, it’s OK if you miss some). 

  | One-on-one mentoring calls | Community calls
-- | -- | --
Purpose | Discuss and strategize: how do you work, identify how to be more efficient, set/review/meet goals | Learn about existing tools & practices and how to engage, meet other labs and see commonalities, break-out groups (split Champions and lab members) to discuss steps forward + keep accountable; have fun!
Outcomes | Increase personal efficiency in at least one concrete way, enable your lab/department, become a Champion for open practices | Increase your lab’s efficiency in several concrete ways, build community and skillsets in your department and beyond
Process | Frequency: 1x per month, 1 hourParticipants: you, JulieMedium: Zoom, Google DocsAdditional possibility: Julie joins one lab meeting per month remotely | Frequency: 2x per month, 1.5 hours Participants: all Champions and their lab members, JulieMedium: Zoom (breakout rooms), Google Docs

**November Intro Call:** 
I’d like to meet with you for an hour with this agenda:

1. Intro to open practices, get you as excited as I am!
1. Understand how you/your lab works (I’ll have prompting questions)
1. Start framing goals for you/your lab (will help me plan)
1. Answer your questions
1. Logistics: 
    - Discuss which lab member to include (get contact info)
    - Schedule repeat one-on-ones, ballpark times for community calls, seminar 

**Next Steps:**
Book a 1-hour intro call with me in November here. Time slots are available across all weeks in November; if none of them work for you please let me know and we’ll figure out another time.

I’m so excited to work with you, and please let me know if you have any questions!

Cheers,
Julie

\* “Lab” is broadly defined; it’s whatever group you work with or will be working with in the future. It can even be Future You.
